### ISSUE TITLE
Running Evaluation on pretrained model

### ISSUE THREAD

**Issue by Akashdeep1000** [akamaster/pytorch_resnet_cifar10#41]
While running evaluate mode directly through command line ">python trainer.py --arch resnet20 --pretrained --evaluate --print-freq 5
Files already downloaded and verified
Test: [0/79]    Time 13.405 (13.405)    Loss 55.9279 (55.9279)  Prec@1 6.250 (6.250)
Test: [5/79]    Time 0.006 (2.239)      Loss 48.1283 (57.7435)  Prec@1 3.906 (5.599)
Test: [10/79]   Time 0.005 (1.224)      Loss 56.3518 (57.2674)  Prec@1 5.469 (6.250)
Test: [15/79]   Time 0.006 (0.843)      Loss 54.6870 (57.2780)  Prec@1 7.031 (6.201)
Test: [20/79]   Time 0.006 (0.644)      Loss 60.2548 (57.5978)  Prec@1 7.031 (5.952)
Test: [25/79]   Time 0.006 (0.521)      Loss 58.7205 (57.5096)  Prec@1 9.375 (6.160)
Test: [30/79]   Time 0.006 (0.438)      Loss 52.3959 (57.5041)  Prec@1 7.031 (6.174)
Test: [35/79]   Time 0.006 (0.378)      Loss 54.9284 (57.7886)  Prec@1 6.250 (6.033)
Test: [40/79]   Time 0.005 (0.333)      Loss 60.2110 (57.8033)  Prec@1 6.250 (6.040)
Test: [45/79]   Time 0.005 (0.297)      Loss 57.6955 (57.9990)  Prec@1 6.250 (6.063)
Test: [50/79]   Time 0.004 (0.268)      Loss 60.3893 (58.1968)  Prec@1 5.469 (6.066)
Test: [55/79]   Time 0.004 (0.245)      Loss 66.5257 (58.4542)  Prec@1 4.688 (5.957)
Test: [60/79]   Time 0.006 (0.225)      Loss 64.6813 (58.2924)  Prec@1 7.812 (6.019)
Test: [65/79]   Time 0.005 (0.209)      Loss 51.3470 (57.9437)  Prec@1 7.031 (6.120)
Test: [70/79]   Time 0.006 (0.194)      Loss 64.6101 (57.9421)  Prec@1 8.594 (6.184)
Test: [75/79]   Time 0.004 (0.182)      Loss 60.6900 (57.9247)  Prec@1 3.125 (6.075)"

 for example the classification rate was not printing out. I tried with different models but still this issue remains
